src/Stan/Report/Html.hs

```diff
@@ -133,6 +139,38 @@ stanAnalysis (analysisToNumbers -> AnalysisNumbers{..}) =
         )
     )
 
+stanSummary analysis AnalysisNumbers{..} =
+    ( divClass "row" (blockP "Summary of the static analysis report")
+    # ul_
+        ( li_ (strong_ "Project health: " # prettyHealth anHealth)
+        # li_ ("The project " # showProjectHealth (toProjectHealth anHealth))
+        # summary
+        )
+    )
+  where
+    showProjectHealth :: ProjectHealth -> Text
+    showProjectHealth = toText . \case
+        Unhealthy    -> "is unhealthy"
+        LowHealth    -> "has low health"
+        MediumHealth -> "has medium health"
+        Healthy      -> "is healthy"
```

Comment:
Need to think on wording to make it more helpful and less blaming 😃 Also we can add some advising